schemas/node.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "NULL",
+  "title": "NODE",
+  "description": "This is the smallest unit of the whole history. It contains content: text.",
+  "type": "object",
+  "properties": {
+    "nodeId": {
+      "description": "The unique identifier for a node",
+      "type": "string",
+      "minLength": 5
+    },
+    "nodeName": {
+      "description": "The name of a specific node.",
+      "type": "string"
+    },
+    "content": {
+      "description": "For now, only text. Is it formatted text?",
+      "type": "string"
+    },
+    "choices": {
+      "description": "The set of possible choices for that particular node. From zero or more.",
+      "type": "array",
+      "items": {
+        "description": "One specific choice that leads to a specific node.",
+        "type": "object", // "$ref": move to other schema
+        "properties": {
+          "locationNodeId": {
+            "description": "The node in which this choice is provided.",
```

Comment:
not sure I understand this, this choice already part of the node(it is part of the choices id), why do we need this?

Reply:
+